### ISSUE TITLE
Clusters Not Displaying

### ISSUE THREAD

**Issue by TonyGnk** [ramani-maps/ramani-maps#111]
The clusters example provided in the repository does not display any points of earthquakes.

[Clusters Example](https://github.com/ramani-maps/ramani-maps/tree/main/examples/clusters/app/src/main/java/org/ramani/example/clusters)

**Steps to Reproduce**

-Clone the repository.
-Navigate to the clusters example directory.
-Build and run the application using the provided source code.
-Observe that no earthquake points are displayed on the map.

**Expected Behavior**

The example should display clustered earthquake points on the map, demonstrating the clustering functionality.

**Actual Behavior**

The map renders, but no earthquake points or clusters are displayed.

**Environment**

- Version: 8.1
- Android version: 15
- Device or Emulator: Pixel 9

**Comment by JonasVautherin** [comment]
Thanks for the report!

It may be fixed on the `main` branch, I'm not sure. I think it's generally related to the other issues with have, because I messed up something with the StyleBuilder. I'm working on that, I'm hoping to have a proper fix in a few days 👍.

**Comment by TonyGnk** [comment]
It appears the issue persists across multiple versions (0.5.0, 0.7.0 and 0.8.1) when using the earthquake data from the provided GeoJSON URL: https://www.mapbox.com/mapbox-gl-js/assets/earthquakes.geojson. However, replacing it with a custom GeoJSON string, such as:

```kotlin
val geoJsonString = """
{
    "type": "FeatureCollection",
    "features": [
        { "type": "Feature", "geometry": { "type": "Point", "coordinates": [-77.032, 38.913] }, "properties": { "mag": 4.5 } },
        { "type": "Feature", "geometry": { "type": "Point", "coordinates": [-122.4194, 37.7749] }, "properties": { "mag": 5.0 } }
    ]
}
"""
val mySource = GeoJsonSource("earthquakes", geoJsonString)
```
...the clustering functionality works in earlier versions up to 0.7.0, but stops functioning in 0.8.0.

**Comment by JonasVautherin** [comment]
Did you manage to try the `main` branch?

Some more changes will come with https://github.com/ramani-maps/ramani-maps/pull/112 but I haven't pushed them yet.

**Comment by TonyGnk** [comment]
Yes, I tried the main branch.

**Comment by JonasVautherin** [comment]
It should be fixed in 0.8.2. Would you mind checking that? 😇
